libs/watt/src/lib/components/modal/+storybook/watt-modal-from-class.component.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, OnInit } from '@angular/core';
+import { WattButtonComponent } from '../../button/watt-button.component';
+import { WATT_MODAL } from '../watt-modal.component';
+import { WattModalService } from '../watt-modal.service';
+import { WattTextFieldComponent } from '../../text-field/watt-text-field.component';
+import { FormControl } from '@angular/forms';
+
+@Component({
+  standalone: true,
+  selector: 'watt-modal-from-class',
+  imports: [WATT_MODAL, WattTextFieldComponent, WattButtonComponent],
+  template: `
+    <watt-modal #modal [title]="title" closeLabel="Close modal" [loading]="isLoading">
+      <watt-text-field [formControl]="exampleFormControl" label="Username" />
```

Comment:
issue: weird but when I type in the input, there's an error in the console.

![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/7aaac63e-5215-4ca5-ad6d-d4a0280d58bc)

Reply:
The error was caused by not importing `ReactiveFormsModule` (good catch!).